docs/notebooks/simulation/particle_stepper.ipynb

```diff
@@ -4,10 +4,7 @@
    "cell_type": "code",
```

Comment:
<p>Can you make this <code>grid</code> a link to the CartesianGrid object in the docs?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PlasmaPy/PlasmaPy/pull/2245/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>